code_search/src/code_search/dataflow/transforms/bigquery.py

```diff
@@ -77,12 +77,10 @@ class BigQueryWrite(beam.PTransform):
     ]
   """
 
-  def __init__(self, project, dataset, table, batch_size=500,
+  def __init__(self, table, batch_size=500,
```

Comment:
Per: https://github.com/kubeflow/examples/issues/381

Rather than make changes to this class; can we just update the calling locations to not use BigQueryWrite?
I will try to identify other change in this PR that motivated this change and provide a helpful comment there.